exec/java-exec/src/main/java/org/apache/drill/exec/planner/sql/parser/SqlCreateView.java

```diff
@@ -47,20 +47,19 @@ public SqlCall createCall(SqlLiteral functionQualifier, SqlParserPos pos, SqlNod
   private SqlIdentifier viewName;
   private SqlNodeList fieldList;
   private SqlNode query;
-  private boolean replaceView;
+  private SqlLiteral createViewType;
 
-  public SqlCreateView(SqlParserPos pos, SqlIdentifier viewName, SqlNodeList fieldList,
-      SqlNode query, SqlLiteral replaceView) {
-    this(pos, viewName, fieldList, query, replaceView.booleanValue());
+  public enum SqlCreateViewType {
+    SIMPLE, ORREPLACE, IFNOTEXISTS
```

Comment:
Please use underscore to separate words: `OR_REPLACE`, `IF_NOT_EXISTS`.

Reply:
done